docusaurus/docs/quickstarts/network/hosted.md

```diff
@@ -20,25 +20,14 @@ the configuration options available you'll be better equipped to make changes.
 
 The first issue you will need to deal with is opening some ports. A network will consist of at least one controller and 
 at least one edge router. Both of these components will require ports to be open. For the controller you will need to 
-open two ports through your firewall, one port for the REST API, and one the control plane. Edge routers will also 
-require two ports open. One for the links created between routers to form the mesh network, and one port for incoming 
-client connections.
+open a range of ports through your firewall:
 
-The ports you choose are not important but unless you change them these ports will default to the following:
+- `8440/tcp`: Edge Controller listening for clients

```

Comment:
8440 doesn't exist. not sure where that came from. it should be 8441 and should state "Edge Controller API"
6262 is the Control plane, I'd just refer to it as that . it's definitely not 8441

8442 is for Edge Connections or the data plane

Reply:
i see you changed this below - i'd rather keep 8441 as the API  use 8440 as the control plane just to minimize the changes

Reply:
- `8440/tcp`: Edge Controller providing router control plane
- `8441/tcp`: Edge Controller providing client sessions
- `8442/tcp`: Edge Router providing client connections
- `8443/tcp`: Ziti Admin Console (ZAC) [optional]